addons/l10n_th/views/report_cash_voucher_templates.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_cash_voucher_document" inherit_id="account.report_payment_receipt_document" primary="True">
+        <xpath expr="//h3" position="before">
+            <!-- Add Company Information -->
+            <t t-set="comp" t-value="o.company_id.partner_id"/>
+            <div class="row">
+                <div class="col-6 offset-6">
+                    <div>
+                        <span t-field="comp.name" /><br/>
+                        <span t-field="comp.vat"/> <span t-esc="comp.l10n_th_branch_number if comp.l10n_th_company_info == 'branch' else 'Headquarter'" /><br/>
```

Comment:
Is not this a bit messy?  Because is every branch having its own company in Odoo then?  Or should you have the possibility on the invoice to tell from which branch it comes?

Reply:
Yes it is required to know which branch it comes from for a company